tomopy/util/mproc.py

```diff
@@ -73,7 +73,11 @@
 SHARED_OUT = None
 SHARED_QUEUE = None
 ON_HOST = False
+DEBUG = False
 
+def set_debug(val=True):
```

Comment:
That's a great feature! Any chance you can explain with few words the purpose of setting ```DEBUG``` here for keeping things clear and readable? These set of global variables are growing and make me worried about how to document about them.